src/Native/Runtime/PalARM64IntrinsicConstants.h

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef PAL_ARM64INTRINSICCONSTANTS_INCLUDED
```

Comment:
Could you please call it just `IntrinsicConstants.h` and move the x86/x64 constants from startup.cpp to this file as well, so that the style is similar between different platforms?